packages/fluentui/react-icons-northstar/src/components/ContentViewGalleryIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { createSvgIcon } from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+export const FormatIcon = createSvgIcon({
```

Comment:
```suggestion
export const ContentViewGalleryIcon = createSvgIcon({
```

It looks that there is a mismatch in naming, can you please check?

Reply:
Good catch, I totally missed that one. Thanks!